ethereum/executionclient/src/main/java/tech/pegasys/teku/ethereum/executionclient/rest/OkHttpRestClient.java

```diff
@@ -118,9 +118,11 @@ private <S> RequestBody createRequestBody(
     return new RequestBody() {
 
       @Override
-      public void writeTo(@NotNull BufferedSink bufferedSink) throws JsonProcessingException {
-        JsonUtil.serializeToBytes(
-            requestBodyObject, requestTypeDefinition, bufferedSink.outputStream());
+      public void writeTo(@NotNull final BufferedSink bufferedSink) throws IOException {
```

Comment:
we don't usually use `@NotNull` annotations so that can probably disappear... not sure we need `bufferedSink;` at the start of the try with resources?

Reply:
we could potentially have a test as well...

Reply:
Intellij does complain, but ignore it... I think it can be disabled in preferences, but I haven't gone looking. We definitely don't want `@NotNull` everywhere..

Reply:
kk cool fixed